english/cuddle-the-cat/cuddle-cat.md

```diff
@@ -3,3 +3,5 @@
 You pick up your kitty and cuddle her for hours. She happily purrs. And
 you happily purr back. Your cat looks at you like you're crazy, but what
 are you going to do?
+
+[Try to rub your cat's belly](/rub-cat-belly/rub-cat-belly.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.